big_tests/tests/sm_SUITE.erl

```diff
@@ -95,6 +94,9 @@ end_per_suite(Config) ->
     escalus_fresh:clean(),
     escalus:end_per_suite(NewConfig1).
 
+init_per_group(parallel, Config) ->
+    register_handler(<<"localhost">>),
```

Comment:
So I can put the same tests repeated within a group and stress-test it (I was running the group with a list built as `lists:flatten([parallel_test_cases() || _ <- lists:seq(1,50)])` to get bigger and bigger feedback that the test is deterministic). It doesn't affect the rest of the group so it doesn't hurt doing that at once.

Reply:
It is still a debugging artefact and should be removed. Placing it in `init_per_group` indicates that is related to the whole group or for some reason cannot be used in `init_per_testcase`. We don't leave lots of `ct:pal("123", [])` around because "they don't hurt".